rust/agama-server/src/agama-web-server.rs

```diff
@@ -103,15 +103,22 @@ impl ServeArgs {
     fn ssl_acceptor(&self) -> Result<SslAcceptor, openssl::error::ErrorStack> {
         let mut tls_builder = SslAcceptor::mozilla_modern_v5(SslMethod::tls_server())?;
 
+        // use default or explicitly provided certificate if any
         if let (Some(cert), Some(key)) = (self.cert.clone(), self.key.clone()) {
             tracing::info!("Loading PEM certificate: {}", cert);
             tls_builder.set_certificate_file(PathBuf::from(cert), SslFiletype::PEM)?;
 
             tracing::info!("Loading PEM key: {}", key);
             tls_builder.set_private_key_file(PathBuf::from(key), SslFiletype::PEM)?;
         } else {
+            tracing::info!("Creating self-signed certificate");
+
+            // create a self-signed certificate if needed and store it for later use
             let (cert, key) = agama_server::cert::create_certificate()?;
 
+            // tries to write generated self-signed certificate. Nobody cares if it fails
```

Comment:
It is not a big issue, but I think it deserves a log entry.

Reply:
sure ... once i merge upstream ;-)